### DISCUSSION TITLE
Comparison of Audio Quality: just_audio, audioplayers, and video_player on Zenfone 8

### DISCUSSION THREAD

**Discussion by Tokenyet** [bluefireteam/audioplayers#1813] [Show and tell]
After experimenting with just_audio, audioplayers, and video_player on my Zenfone 8 (Android), I noticed significant differences in music quality across these packages. It's clear that just_audio offers the poorest music quality, while audioplayers performs better but isn't the best. Surprisingly, video_player delivers the highest music quality among the three.

I'm sharing this information because audioplayers appears to be the most well-maintained third-party audio player. I hope this insight proves useful to others in the community.

**Comment by satvikpendem** [comment]
Is this still accurate?
